Sources/XcbeautifyLib/CaptureGroups.swift

```diff
@@ -84,7 +84,7 @@ struct CodesignFrameworkCaptureGroup: CaptureGroup {
 
 struct CompileCaptureGroup: CompileFileCaptureGroup {
 #if !os(Linux)
-    let filePath: String
+    let wholeError: String
```

Comment:
Why did you change this to `wholeError`? That seems unexpected based on the type. We already have a `CompileErrorCaptureGroup` that we should use for compiler errors.

Reply:
It is renamed by Xcode.

That should be reverted.